src/licensedcode/data/licenses/oil-1.0.yml

```diff
@@ -0,0 +1,14 @@
+key: oil-1.0
+short_name: OIL-1.0
+name: Open Innovation License version 1.0
+category: Permissive
+owner: Stark Drones Corporation
+notes: From https://github.com/StarkDrones/OPNLv1/blob/main/.github/LICENSE.md
+homepage_url: https://www.en.patent.aau.dk/open-innovation-licensing/
```

Comment:
I do not think this is the home page and that this Danish outfit is related to this license here. They vaguely use similar names. Or may be I am missing something?